modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,289 @@
+:description: Page contains lists of procedures, configuration settings, metrics removed or deprecated in Neo4j 2025. Also, you can find information on changed defaults and new functionality of neo4j-admin commands.
+
+//Check Mark
+:check-mark: icon:check[]
+
+
+[[removals-deprecations-2025-01]]
+= Deprecations, removals, and changes per Neo4j version
+
+== Neo4j 2025.01
+
+=== Removed built-in Neo4j procedures
+
+.See procedures removed in Neo4j 2025.01 without replacement.
+[%collapsible]
+====
+[options=header,cols="3m,1,1"]
+|===
+| Name
+| Community Edition
+| Enterprise Edition
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_cluster_movetonextdiscoveryversion[`dbms.cluster.moveToNextDiscoveryVersion()`]
+|
+| {check-mark}
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_cluster_showparalleldiscoverystate[`dbms.cluster.showParallelDiscoveryState()`]
+|
+| {check-mark}
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_cluster_switchdiscoveryserviceversion[`dbms.cluster.switchDiscoveryServiceVersion()`]
+|
+| {check-mark}
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_setDatabaseAllocator[`dbms.setDatabaseAllocator()`]
+|
+|{check-mark}
+
+|===
+====
+
+=== Removed Neo4j configuration settings
+
+.See all removed configuration settings in Neo4j 2025.01
+[%collapsible]
+====
+[options=header,cols="4m,2"]
+|===
+|Name
+|Notes
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings/#config_db.cluster.raft.leader_transfer.priority_group[`db.cluster.raft.leader_transfer.priority_group`]
+|label:enterprise[Enterprise Edition]
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_db.logs.query.annotation_data_as_json_enabled[`db.logs.query.annotation_data_as_json_enabled`]
+|label:dynamic[]
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_db.tx_state.memory_allocation[`db.tx_state.memory_allocation`]
+|
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_dbms.cluster.catchup.client_inactivity_timeout[`dbms.cluster.catchup.client_inactivity_timeout`]
+|label:enterprise[Enterprise Edition]
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_dbms.cluster.discovery.log_level[`dbms.cluster.discovery.log_level`]
+|label:enterprise[Enterprise Edition]
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_dbms.cluster.discovery.type[`dbms.cluster.discovery.type`]
+|label:enterprise[Enterprise Edition]
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_dbms.cluster.discovery.v2.endpoints[`dbms.cluster.discovery.v2.endpoints`]
```

Comment:
not removed, but renamed? cc @nick-giles-neo

Reply:
This one is deprecated.